recipes/earthpy/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "earthpy" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f604ffe3666d59f4104bff2ca887312c4c7b827a87f8716561ae86b5e9b76a05
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - geopandas
+    - matplotlib >=2
+    - numpy
+    - pandas
+    - rasterio
+    - Rtree>=0.8
```

Some of the automated services fail if you don't include the space here:
```suggestion
    - Rtree >=0.8
```

---

@mbjoseph do we need to update our setup.py to accommodate this as well?

---

Nope, it only affect's conda-forge's ability to parse the `meta.yaml` file.

---

Thanks Chris!